pyTigerGraph/pyTGCyto.py

```diff
@@ -0,0 +1,61 @@
+import pyTigerGraph
+
+class pyTGCyto: 
```

Comment:
Could you give us a sentence or to about why we should add this module? I personally don't know what Cyto is.